src/models/links.cr

```diff
@@ -1,6 +1,6 @@
 # These are external URLs we use throughout the site
 module Links
   def self.chat_url
-    "https://discord.gg/HeqJUcb"
```

Comment:
lol I love how we had this and barely used it 😂

Reply:
Oops, yeah I need to remove this module now!

I didn't realize I'd removed every link.